net-libs/libwebsockets/libwebsockets-4.0.4.ebuild

```diff
@@ -12,14 +12,16 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="MIT"
 SLOT="0/16" # libwebsockets.so.16
 KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
You should drop keywords and request re-keywording when you introduce a new dependency (mbedTLS here).